### PR TITLE
Add Kc-Align tool

### PR DESCRIPTION
**Pull request by nickeener**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/kc-align/kc-align.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<tool id="kc-align" name="Kc-Align" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package" version="0.5">kcalign</requirement>
+        <requirement type="package" version="2.04">kalign2</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+        <![CDATA[
+        kc-align
+            --mode $position.mode
+            --reference $reference
```
Comment (bgruening) [inline tools/kc-align/kc-align.xml]: please add single-quotes

File: tools/kc-align/kc-align.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        <param name="mode" value="genome" />
+        <param name="start" value="21563" />
+        <param name="end" value="25384" />
+        <output name="fasta" ftype="fasta" compare="diff" value="kc-align.fasta" />
+        <output name="clustal" ftype="txt" compare="diff" value="kc-align.clustal" />
+    </test>
+</tests>
+    <help><![CDATA[
+
+        Kc-Align Help
```
Comment (bgruening) [inline tools/kc-align/kc-align.xml]: can this be removed?

File: tools/kc-align/kc-align.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+        Kc-Algin is a codon-aware multiple aligner that uses Kalgin2 to produce in-frame gapped codon alignments for selection analysis of small genomes (mostly viral and some smaller bacterial genomes). Takes nucleotide seqeunces as inputs, converts them to their in-frame amino acid sequences, performs multiple alignment with Kalign, and then converts the alignments back to their original codon sequence while preserving the gaps. Produces two outputs: the gapped nucleotide alignments in FASTA format and in CLUSTAL format.
+
```
Comment (bgruening) [inline tools/kc-align/kc-align.xml]: You can use http://rst.ninjs.org to tweak the help a little bit.

Comment (nickeener) [reply tools/kc-align/kc-align.xml]: Is there a standard format to follow for the help section?

Comment (bgruening) [reply tools/kc-align/kc-align.xml]: Not really, but please take inspiration from other tools here.

**Comment by nickeener** [comment]
@bgruening Can you check the modified help section and the .shed.yml file and let me know if anything else needs changed? Thanks.

**Comment by bgruening** [comment]
Looks good to me. Ping @davebx!

**Comment by nickeener** [comment]
@davebx Can you please review this and merge if it's good. Thanks!

**Pull request merged**